utils/create_test_avatar.py

```diff
@@ -0,0 +1,20 @@
+import tempfile
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from PIL import Image
+
+from core.models import User
+
+
+def create_test_avatar(user: User) -> None:
```

Comment:
Where is assert in this test?

Reply:
This is not a test but an additional function and saves the tagged image maybe I called it wrong

Reply:
definetely :-)